packages/react/src/components/LinkList/__stories__/LinkList.stories.js

```diff
@@ -70,6 +70,7 @@ Default.story = {
               src: '0_uka1msg4',
               type: 'video',
             },
+            disableImage: true,
```

Comment:
@annawen1 I wonder if LinkList should disable the video and not have it set in the story? we probably can't rely on the adopter to set this themselves.

Reply:
hmm good point, yeah I'll set the prop in the component itself